### PR TITLE
[Reviewer: AJL] Split storage debian packaging for Homer/Homestead-prov

### PR DESCRIPTION
**Pull request by sathiyan-sivathas**
(6/6)

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -15,11 +15,21 @@ Depends: clearwater-infrastructure, clearwater-nginx, libxml2-dev, libxslt1-dev,
...
+Architecture: any
+Depends: homer
+Description: homer-node, the Cassandra powered XDMS
+
 Package: homer
 Architecture: any
 Depends: crest
 Description: homer, the Cassandra powered XDMS
 
+Package: homestead-node-prov
```
Comment (johadalin) [inline debian/control]: So having a homestead-node-prov package doesn't really make much sense, as we can't really deploy just homestead-prov on a node on its own. 
Instead, i think we should just keep it as the single homestead-prov package, and have it as a dependency of the homestead-node package, as we always say that both should be installed at the same time anyway

**Comment by johadalin** [comment]
One comment, otherwise fine

**Comment by johadalin** [comment]
Looks fine. 

I have just noticed something i missed in my design though. 
Homestead-prov depends on crest, which in turn depends on clearwater-cassandra

@graemerobertson or @AlexHockey , are you able to give any thoughts on this? 
I had a look at the design for the cassandra schema work, but it isn't clear if we are going to be removing homestead-prov from homestead nodes in the new-world, or reworking the -prov package to remove dependency on cassandra. 

It may make sense to talk about this at some point today.

**Comment by AlexHockey** [comment]
I think:

* None of homestead, homestead-prov, homer, or crest shouldn't depend on clearwater-cassandra. 
* homestead-node should bring in clearwater-cassandra, homestead and homestead-prov.
* homer-node should bring in clearwater-cassandra and homer 

Does this make sense (and work :-)?

**Comment by graemerobertson** [comment]
"None of homestead, homestead-prov, homer, or crest shouldn't depend on clearwater-cassandra."

"Does this make sense?" No

**Comment by sathiyan-sivathas** [comment]
I'm confused as well, was that an intentional double negative?  I'm guessing not?

**Comment by AlexHockey** [comment]
I've edited so my comment makes sense now. Sorry!

**Comment by johadalin** [comment]
Will Homer run fine without cassandra in a split-storage deployment?

Other than that query, your suggestiong seems like a reasonable plan

**Comment by johadalin** [comment]
As discussed with @AlexHockey, we can resolve the issues of homer in a split-storage deployment with some further work elsewhere. 
@sathiyan-sivathas, can you make the dependency changes as above, separating the clearwater-cassandra dependency out of Crest, and pulling it in where it's needed? 
Let me know if anything is unclear :)

**Pull request merged**